src/utils/template_handler.rs

```diff
@@ -200,6 +259,52 @@ pub(crate) fn load_remote_template(path: &str, url: &str, force: bool) -> Status
         }
     }
 
+    return Status::ok();
+}
+
+/// Load a template from a remote repository
+pub(crate) fn load_remote_template(
+    path: &str,
+    url: &str,
+    force: bool,
+    url_type: &URLType,
```

Comment:
Same here with the type. Maybe add a internal function that is checking the type to avoid repetition of the check